navigating to the wallet info page during a rescan causes the app to crash

see crash log below:

```
2023-10-06 11:33:23.563 [INF] DCR: Rescanning block range [1, 1999]...
2023-10-06 11:33:24.294 [INF] DCR: Rescanning block range [2000, 3999]...
2023-10-06 11:33:24.603 [INF] DCR: Rescanning block range [4000, 5999]...
2023-10-06 11:33:25.049 [INF] DCR: Rescanning block range [6000, 7999]...
2023-10-06 11:33:25.730 [INF] DCR: Rescanning block range [8000, 9999]...
2023-10-06 11:33:26.531 [INF] DCR: Rescanning block range [10000, 11999]...
2023-10-06 11:33:27.151 [INF] DCR: Rescanning block range [12000, 13999]...
2023-10-06 11:33:27.729 [INF] DCR: Rescanning block range [14000, 15999]...
2023-10-06 11:33:28.622 [INF] DCR: Rescanning block range [16000, 17999]...
2023-10-06 11:33:29.197 [INF] DCR: Rescanning block range [18000, 19999]...
2023-10-06 11:33:30.048 [INF] DCR: Rescanning block range [20000, 21999]...
2023-10-06 11:33:30.656 [INF] DCR: Rescanning block range [22000, 23999]...
2023-10-06 11:33:31.234 [INF] DCR: Rescanning block range [24000, 25999]...
2023-10-06 11:33:32.090 [INF] DCR: Rescanning block range [26000, 27999]...
2023-10-06 11:33:33.424 [INF] DCR: Rescanning block range [28000, 29999]...
2023-10-06 11:33:34.110 [INF] DCR: Rescanning block range [30000, 31999]...
2023-10-06 11:33:34.811 [INF] DCR: Rescanning block range [32000, 33999]...
2023-10-06 11:33:35.596 [INF] DCR: Rescanning block range [34000, 35999]...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x118 pc=0x130a83d]

goroutine 31 [running]:
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout.func1.1.1.1.1({{{0x0, 0x0}, {0x1e8, 0xf412c}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:297 +0x3d
gioui.org/layout.Inset.Layout({0x176?, 0x0?, 0x7d5e?, 0x0?}, {{{0x0, 0x0}, {0x1e8, 0xf412c}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout.func1.1.1.1({{{0x0, 0x0}, {0x1e8, 0xf4136}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:296 +0xc9
gioui.org/layout.Flex.Layout({0x60?, 0x1?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x1e8, 0xf4136}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout.func1.1.1({{{0x0, 0x0}, {0x1e8, 0xf4136}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:294 +0x3f5
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x1e8, 0xf4136}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/components.Container.Layout(...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/components/components.go:82
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout.func1.1({{{0x0, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:293 +0x149
gioui.org/layout.Stack.Layout({0x0?}, {{{0x208, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout.func1({{{0x208, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:64 +0x18e
gioui.org/layout.Inset.Layout({0x1742940?, 0x0?, 0x12?, 0x0?}, {{{0x208, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout({{0x0, 0x0, 0x0, 0x0}, {0xf3, 0xf5, 0xf6, 0xff}, {0xf3, 0xf5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:63 +0x10b
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout.func1({{{0x208, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:292 +0x2fd
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x208, 0xf4156}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).rescanDetailsLayout(0xc0032e86c0, {{{0x0, 0x0}, {0x208, 0xf4160}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:288 +0x212
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).syncStatusSection.func1.1.3({{{0x0, 0x0}, {0x208, 0xf4160}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:55 +0x85
gioui.org/layout.Flex.Layout({0x80?, 0x1?, 0x70?, 0xc0?}, {{{0x0, 0x0}, {0x208, 0xf41ae}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).syncStatusSection.func1.1({{{0x0, 0x0}, {0x208, 0xf41ae}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:37 +0x2fb
gioui.org/layout.Inset.Layout({0x3f800000?, 0x3f800000?, 0x1f7cd40?, 0x0?}, {{{0x0, 0x0}, {0x208, 0xf41ae}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/components.Container.Layout(...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/components/components.go:82
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).syncStatusSection.func1({{{0x0, 0x0}, {0x208, 0xf41cd}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:36 +0x134
gioui.org/layout.Stack.Layout({0x0?}, {{{0x0, 0x0}, {0x208, 0xf41cd}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout.func1({{{0x0, 0x0}, {0x208, 0xf41cd}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:64 +0x18e
gioui.org/layout.Inset.Layout({0x1742cc0?, 0x0?, 0x3b1cb30?, 0xc0?}, {{{0x0, 0x0}, {0x208, 0xf41cd}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout({{0x0, 0x0, 0x0, 0x0}, {0xff, 0xff, 0xff, 0xff}, {0xf3, 0xf5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:63 +0x10b
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).syncStatusSection(0xc0032e86c0, {{{0x0, 0x0}, {0x208, 0xf41cd}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/sync_status.go:32 +0x2d8
gioui.org/layout.Flex.Layout({0x56?, 0x66?, 0x57?, 0x0?}, {{{0x0, 0x0}, {0x208, 0xf4218}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout.func1.1.1.1.1({{{0x0, 0x0}, {0x208, 0xf4218}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:120 +0x255
gioui.org/layout.Inset.Layout({0x326b680?, 0xc0?, 0x12b?, 0x0?}, {{{0x0, 0x0}, {0x208, 0xf4218}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout.func1.1.1.1({{{0x0, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:119 +0x89
gioui.org/layout.Stack.Layout({0xb4?}, {{{0x230, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout.func1({{{0x230, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:64 +0x18e
gioui.org/layout.Inset.Layout({0x1665360?, 0x0?, 0x14ecc8?, 0xc0?}, {{{0x230, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.Card.Layout({{0x0, 0x0, 0x0, 0x0}, {0xff, 0xff, 0xff, 0xff}, {0xf3, 0xf5, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/card.go:63 +0x10b
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout.func1.1.1({{{0x230, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:118 +0x17f
gioui.org/layout.Inset.Layout({0x326b680?, 0xc0?, 0x12a?, 0x0?}, {{{0x230, 0x0}, {0x230, 0xf4240}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout.func1.1({{{0x232, 0x0}, {0x232, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:117 +0x89
gioui.org/layout.(*List).Layout(0xc0032703d0, {{{0x232, 0x0}, {0x232, 0xf4240}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/list.go:122 +0x17c
gioui.org/widget/material.ListStyle.Layout({0xc003270380, {0xc003270380, {0x40000000, 0x40000000, {0x0, 0x0, 0x0, 0x0}}, {0x42180000, 0x40c00000, ...}}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/widget/material/list.go:290 +0x20c
github.com/crypto-power/cryptopower/ui/cryptomaterial.ListStyle.Layout(...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/list_scroll.go:39
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout.func1({{{0x0, 0x0}, {0x23c, 0x1cc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:116 +0x185
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x23c, 0x1cc}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/page/components.UniformPadding({{{0x0, 0x0}, {0x26c, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/components/components.go:100 +0x197
github.com/crypto-power/cryptopower/ui/page/info.(*WalletInfo).Layout(0xc00326b680?, {{{0x0, 0x0}, {0x26c, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/info/info_page.go:158 +0x65
github.com/crypto-power/cryptopower/ui/page/root.(*MainPage).layoutDesktop.func1.1.2({{{0x0, 0x0}, {0x26c, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/main_page.go:652 +0x1d9
gioui.org/layout.Flex.Layout({0xb7?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x320, 0x1fc}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x0?, 0x0?}, {{{0x320, 0x1fc}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x320, 0x1fc}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x320, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x320, 0x1fc}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0x57?}, {{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x63e5b7aa?, 0x8?, 0x2e7a220?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x0, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/page/root.(*MainPage).layoutDesktop.func1.1({{{0x0, 0x0}, {0x320, 0x1fc}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/main_page.go:625 +0x20b
gioui.org/layout.Flex.Layout({0x8c?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:99 +0x2d5
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x320, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x0?, 0x0?, 0x41c00000?, 0x0?}, {{{0x320, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x320, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x320, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x320, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0xad?}, {{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x29e5?, 0x0?, 0xa5?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x1, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/page/root.(*MainPage).layoutDesktop.func1({{{0x0, 0x0}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/main_page.go:618 +0x20b
gioui.org/layout.Stack.Layout({0x2?}, {{{0x0, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:78 +0x498
github.com/crypto-power/cryptopower/ui/page/root.(*MainPage).layoutDesktop(0xc00621d200, {{{0x0, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/main_page.go:612 +0x12b
github.com/crypto-power/cryptopower/ui/page/root.(*MainPage).Layout(0xc00621d200, {{{0x0, 0x227}, {0x320, 0x227}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/main_page.go:608 +0x105
gioui.org/layout.Flex.Layout({0x5?, 0x0?, 0x0?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/flex.go:134 +0x738
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1.1({{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:88 +0x85
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1.1({{{0x320, 0x28a}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:87 +0xe5
gioui.org/layout.Inset.Layout({0x3b23ac0?, 0xc0?, 0x6b3eb7?, 0x0?}, {{{0x320, 0x28a}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2.1({{{0x320, 0x28a}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:85 +0x109
github.com/crypto-power/cryptopower/ui/cryptomaterial.Border.Layout({{0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0x0, 0x0}, 0x0}, {{{0x320, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/border.go:21 +0x98
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1.2({{{0x320, 0x28a}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:83 +0x1d6
gioui.org/layout.Stack.Layout({0x0?}, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:60 +0x195
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1.1({{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:46 +0x2aa
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1.1({{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:105 +0x238
gioui.org/layout.Inset.Layout({0x47a607?, 0x0?, 0x576c73?, 0x0?}, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:169 +0x39d
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout.func1({{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:44 +0x109
gioui.org/layout.Direction.Layout(0x0, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/layout.go:196 +0x162
github.com/crypto-power/cryptopower/ui/cryptomaterial.LinearLayout.Layout({0xfffffffffffffffe, 0xfffffffffffffffe, 0x1, {0x0, 0x0, 0x0, 0x0}, 0x0, {{0x0, 0x0, ...}, ...}, ...}, ...)
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/cryptomaterial/linearlayout.go:42 +0xe5
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).layoutDesktop.func1({{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, 0x863e5b7aa, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:340 +0x425
gioui.org/layout.Stack.Layout({0x4c?}, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...}, ...)
        /home/kennedy/Projects/pkg/mod/gioui.org@v0.1.0/layout/stack.go:78 +0x498
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).layoutDesktop(0xc000004c00, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:334 +0x12b
github.com/crypto-power/cryptopower/ui/page/root.(*HomePage).Layout(0xc000004c00, {{{0x0, 0x0}, {0x320, 0x28a}}, {0x3f800000, 0x3f800000}, {0x1f7cd40, 0xc00014ecc8}, {0xc1402f1c120fc059, ...}, ...})
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/ui/page/root/home_page.go:330 +0x105
created by main.main
        /home/kennedy/Projects/src/github.com/cryptopower/cryptopower/main.go:108 +0x61b
exit status 2
```

--- a/libwallet/assets/btc/rescan.go
+++ b/libwallet/assets/btc/rescan.go
@@ -353,6 +353,7 @@ func (asset *Asset) updateRescanProgress(progress *chain.RescanProgress) {
 	rescanProgressReport := &sharedW.HeadersRescanProgressReport{
 		CurrentRescanHeight: progress.Height,
 		TotalHeadersToScan:  int32(allHeadersToFetch),
+		WalletID:            asset.ID,
 	}
 
 	elapsedRescanTime := time.Now().Unix() - asset.syncData.rescanStartTime.Unix()

--- a/libwallet/assets/dcr/rescan.go
+++ b/libwallet/assets/dcr/rescan.go
@@ -61,6 +61,7 @@ func (asset *Asset) RescanBlocksFromHeight(startHeight int32) error {
 			rescanProgressReport := &sharedW.HeadersRescanProgressReport{
 				CurrentRescanHeight: p.ScannedThrough,
 				TotalHeadersToScan:  asset.GetBestBlockHeight(),
+				WalletID:            asset.ID,
 			}
 
 			elapsedRescanTime := time.Now().Unix() - rescanStartTime

--- a/libwallet/assets/ltc/rescan.go
+++ b/libwallet/assets/ltc/rescan.go
@@ -308,6 +308,7 @@ func (asset *Asset) updateRescanProgress(progress *chain.RescanProgress) {
 	rescanProgressReport := &sharedW.HeadersRescanProgressReport{
 		CurrentRescanHeight: progress.Height,
 		TotalHeadersToScan:  int32(allHeadersToFetch),
+		WalletID:            asset.ID,
 	}
 
 	elapsedRescanTime := time.Now().Unix() - asset.syncData.rescanStartTime.Unix()

--- a/libwallet/assets/wallet/types.go
+++ b/libwallet/assets/wallet/types.go
@@ -273,6 +273,7 @@ type HeadersRescanProgressReport struct {
 	CurrentRescanHeight int32 `json:"currentRescanHeight"`
 	RescanProgress      int32 `json:"rescanProgress"`
 	RescanTimeRemaining int64 `json:"rescanTimeRemaining"`
+	WalletID            int   `json:"walletID"`
 }
 
 type DebugInfo struct {

--- a/listeners/blockrescanprogress.go
+++ b/listeners/blockrescanprogress.go
@@ -30,6 +30,7 @@ func (br *BlocksRescanProgressListener) OnBlocksRescanProgress(progress *sharedW
 	br.UpdateNotification(wallet.RescanUpdate{
 		Stage:          wallet.RescanProgress,
 		ProgressReport: progress,
+		WalletID:       progress.WalletID,
 	})
 }
 

